src/constants.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+    CACHE_DURATION_IMPLEMENTS_PALLET: 1000 * 60 * 60 * 24, // how long to remember if a chain implements the identity pallet
+    ERROR_MESSAGE_IS_ARCHIVED_NODE: "Provided node is not an archive node.",
```

Comment:
@sosan-neikbeen ```ERROR_MESSAGE_IS_ARCHIVED_NODE``` should be ```ERROR_MESSAGE_IS_NOT_ARCHIVED_NODE```

Reply:
@sosan-neikbeen For better understanding we should give the error key the prefix: SERVER_ERROR_...